frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2026,6 +2026,10 @@ function ReaderFooter:onUpdateFooter(force_repaint, full_repaint)
     end
 end
 
+function ReaderFooter:onUseReferencePageUpdate()
```

Comment:
Event handlers are located at the end of the module.
I would try
`ReaderFooter.onUseReferencePageUpdate = ReaderFooter.maybeUpdateFooter`